test/test.py

```diff
@@ -0,0 +1,46 @@
+import subprocess
+import argparse
+import os
+
+AWS_PROFILE = os.environ["AWS_PROFILE"]
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--slower", action="store_true", default=False)
+args = parser.parse_args()
+if args.slower:
+    regions = [
+        "us-east-1",
```

Comment:
We can probably stand to drop most of these regions. Testing in `us-east-1`, `us-west-2`, and perhaps a few select global regions should be fine. 

Alternatively, `aws ec2 describe-regions` should be used if the intent is to test all regions, but that seems unnecessary. The differences between regions shouldn't be meaningful in most cases.

Reply:
So far, I am noticing failures in different regions because some services aren't available in all regions. For example, RDS isn't in all regions.

I think it's valuable to be able to tell customers that our product works in all regions because we tested in all regions. We can't do that yet.

If we know it doesn't work in all regions, then we'd need to do back-and-forth communication, which is costly if it involves manual testing.

I am hoping this can become a CI test that takes ~30 minutes. However, it may be too expensive given how GitHub actions works.

Reply:
Ok, observed inter-region differences is a good reason to test across regions. Do make sure that's programmatic though, and not hardcoded. You can hardcode regions that should be excluded if there's known incompatibilities.

Reply:
As far as Actions costs, the cost of a broken module is way higher than actions minutes, so don't worry about that. What I would worry about is whether someone has to wait 30 minutes to merge a trivial change. Often folks will use scheduled (nightly) builds for this purpose, so that regional edge cases are caught eventually, but development isn't slowed down everywhere to protect against a rare case. You can also offer a `workflow_dispatch` trigger and run the full suite for any potentially risky change that you identify as needing a full test run.